src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -13,13 +13,13 @@ const PopupWithForm = ({
   handleModalClose,
   handleLoginModal,
 
-  isOpen,
   linkToRegOrLogin,
   onSubmit,
   handleRegisterModal,
+  emailNotFoundError,
 }) => {
   return (
-    <div className={`popupwithform popupwithform__type_${name}`}>
```

Comment:
**Could be improved**



Every modal window should have a common class `modal` (or `popup`):

```jsx
    <div className={`modal  moda_type_${name}`}>


```

The idea why all common elements should have **common** css classes is that you can create common `css` styles, common **universal** functions, find all common elements in the DOM and so on…

Also there should be common classes for `cross` button, `form`, `input`, `submit` button and `overlay`